brianlong/www.validators.app
# 181204864_assign_correct_geo_data

#### What's this PR do?
- Create new tables and moves data

#### How should this be manually tested?
- Run migrations
- Run script `rails r script/migrate_from_ips_to_data_centers/copy_data_to_data_centers.rb`
- Check file in `tmp/copy_data_to_data_centers.txt`
- Get one `Ip` and find `ValidatorIp` with the same address, then check `DataCenterHost` and `DataCenter` connected with ValidatorIp, compare with Ip data

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181204864

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
